src/App.vue

```diff
@@ -4,12 +4,23 @@ import { defineComponent } from 'vue';
 import { mapState, mapActions } from 'pinia';
 
 import NavBar from '@/components/global/NavBar/NavBar.vue';
+import DataLoader from '@/components/global/DataLoader.vue';
+import ErrorBoundary from '@/components/global/ErrorBoundary.vue';
 
 import { useMoviesStore } from '@/store/movies.js';
 
 export default defineComponent({
   components: {
-    NavBar
+    NavBar,
+    DataLoader,
+    ErrorBoundary
+  },
+  data() {
+    return {
+      error: null,
+      errorInstance: null,
+      errorInfo: ''
+    };
```

Comment:
not used anymore?

Reply:
Yup, missed to delete these properties - sorry!